src/VirtualMonoRepo/source-mappings.json

```
@@ -105,6 +105,10 @@
                 "src/SourceBuild/content/**/*"
             ]
         },
+        {
+            "name": "llvm-project",
+            "defaultRemote": "https://github.com/dotnet/llvm-project"
+        },
```

```suggestion
        {
            "name": "llvm-project",
            "defaultRemote": "https://github.com/dotnet/llvm-project",
            "exclude": [
                "bolt",
                "clang/docs",
                "clang/www",
                "flang",
                "libclc",
                "lldb",
                "llvm/docs",
                "mlir",
                "openmp",
                "polly",
                "pstl",
                "third-party",
                "**/tests/**"
            ]
        },
```

@tmds seems like with these cloakings, the `git apply` error does not manifest. I believe git might have problems with putting one of the cloaked files into a `.patch` and then using that.

So while I'd still take time to finalize the cloakings, you can apply this change here and the PR build will sync the repo for you and run the build so you can keep working on this while I verify what else needs cloaking.